portfolio/src/main/webapp/style.css

```diff
@@ -39,3 +39,26 @@
   background-color: whitesmoke;
   visibility: visible;
 }
+
+#spongebob-container {
```

Comment:
best practice is to keep these all alphabetized

Reply:
Is it more convenient to alphabetize in chunks by feature or just alphabetize all of the parts as a whole?

Reply:
alphabetize this file independent of the HTML, that way if you change the HTML you don't need to keep them in sync.

Reply:
to be clear - i would put "bubbles" at the top and "spongebob-container" at the bottom. i know you have them grouped together here because they're grouped together in the html, but what if you redid your whole website design to put the spongebob section at the top of the page? if the css classes are just alphabetized, then you don't have to worry about what the HTML looks like in order to find the class you are looking for in this file.